xapian-core/tests/api_closedb.cc

```diff
@@ -242,6 +299,20 @@ DEFINE_TESTCASE(closedb5, transactions) {
 	Xapian::Database db = get_writable_database_as_database();
 	TEST_EQUAL(db.get_doccount(), 0);
     }
+
```

Comment:
The additions below don't really fit the current description of this testcase: "If a transaction is active, close() shouldn't implicitly commit()."

I would suggest they'd fit better in closedb7 (description: "Test WritableDatabase methods") where there's even an existing comment that discusses how `begin_transaction()` should behave, except the condition on closedb7 doesn't include `transactions`.

I think probably move the "If a transaction is active, close() shouldn't implicitly commit()." comment to the first block, add a more generic comment for the testcase such as "Test the effects of close() on transactions" and relocate the part of the comment in closedb7 that talks about `begin_transaction()` to the comment block below.

Reply:
Addressed the changes suggested for comments. Let me know if all ok :)

Reply:
Sorry about the style error. I keep forgetting to run style check script locally.  Will take care of this in future.